### ISSUE TITLE
Onboarding system for a package to be part of the unhcrverse

### ISSUE THREAD

**Issue by dickoa** [unhcRverse/unhcrverse#3]
Hi,

We need to discuss what make a package suitable for the {unhrcverse}. We need to clarifiy it. 
In terms of scope, when does it fit? And when it comes to the quality of the package, how do we assess it ?
Let's say we have a peer review system, do we have enough people to make it work?

**Comment by Edouard-Legoupil** [comment]
Indeed!! key discussion - maybe then content to add here: https://unhcrverse.github.io/unhcrverse/articles/ecosystem.html - Not sure the  "[cathedral](https://en.wikipedia.org/wiki/The_Cathedral_and_the_Bazaar)" approach can fit our set up...  would rather view it as "do what we can" - for instance every 2 months we discuss the existing packages - with a consensus to be obtained in order to get stuff in the unhcrverse.. I pushed now a first iteration to launch discussion - we can definitely review and adjust

**Comment by Edouard-Legoupil** [comment]
for instance - happy to split, as discussed, https://edouard-legoupil.github.io/unhcrdatapackage/ into a standalone chart library on one side and a "data-only" package on the other one..  - but what would be the guiding principle behind this?  - maybe it's purely about __scoping__ ?  - as we could extend the chart library to other corporate unhcr open data sources such as https://unhcr-americas.github.io/iati/docs/

**Comment by dickoa** [comment]
We need to have some type of chat to discuss all of this, right? I totally agree with having it open and discuss.
Should we create a new channel on Teams? using something else?
